src/components/atoms/Button/styles.css

```diff
@@ -0,0 +1,25 @@
+.button {
+    background-color: #0157ff;
+    width: 174px;
+    height: 30px;
+    display: flex;
+    justify-content: center;
+    border-radius: 4px;
+    border: none;
+    outline: none;
+}
+
+.button:active {
+    background-color: #005ec4;
+    transition-duration: 0.2s;
+}
+
+.buttonText {
+    color: white;
+    font-size: 14px;
+    height: 100%;
+    vertical-align: middle; 
```

Comment:
I think it's not needed this property. You can just add ``align-items: center`` in ``.button`` class

Reply:
Then, you should remove the ``height`` property